mineinabyss-components/src/main/kotlin/com/mineinabyss/components/gondolas/Gondola.kt

```diff
@@ -10,7 +10,14 @@ import org.bukkit.Location
 @SerialName("mineinabyss:gondola")
 class Gondola(
     @Serializable(with = LocationSerializer::class)
-    val location: Location,
-    val name: String,
-    val displayItem: SerializableItemStack,
-)
+    val upperLoc: Location,
+    @Serializable(with = LocationSerializer::class)
+    val lowerLoc: Location,
+    val name: String, // the name of the gondola
```

Comment:
can we change this to be an id and add a displayname? so chat would show a displayname but internally its just an id
the config would be a map here not a list

so id would be transient
similar to how packy does it [here](https://github.com/MineInAbyss/Packy/blob/master/src/main/kotlin/com/mineinabyss/packy/config/PackyTemplate.kt)